docker/Dockerfile.base

```diff
@@ -21,18 +21,18 @@
 # You should have received a copy of the GNU General Public License version 2 along with
 # OpenFV. If not, see https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html.
 
-FROM ubuntu:14.04
+FROM nvidia/cuda:10.2-devel-ubuntu16.04
```

Comment:
16.04 is a bit old now. Would it be possible to test if we can bump this up to 20.04? Or at least 18.04.

Just think it would be good to have something that is more future proof.

Reply:
We can go to 18.04 pretty easily, going to 20.04 requires going to CUDA 11, which looks to be a bigger lift.

Reply:
Maybe this should happen after we have some test cases though?

Reply:
Yes CUDA 11 will be tricky. If going to 18.04 isn't hard maybe we should do that?